VoxelSniperForge/src/main/java/com/github/kevindagame/voxelsniperforge/blockstate/sign/ForgeSign.java

```diff
@@ -18,18 +23,35 @@ public ForgeSign(ForgeBlock block, BlockState blockState, SignBlockEntity tileEn
         this.tileEntity = tileEntity;
     }
 
-    @NotNull
-    private static String translateColorCodesToAmpersand(@NotNull String textToTranslate) {
-        throw new UnsupportedOperationException("Not implemented yet");
+    private SignText front() {
+        return tileEntity.getFrontText();
```

Comment:
This method has the same name as the next, but it does something different?

Reply:
I inlined both functions